<?xml version="1.0"?>
<launch>

  <arg name="config_file" description="The config file that provides important parameters for setting up the adapter"/>
  <arg name="nav_graph_file" description="The graph that this fleet should use for navigation"/>
  <arg name="server_uri" default="" description="The URI of the api server to transmit state and task information."/>
  <arg name="output" default="screen"/>

  <!-- Fleet adapter -->
  <node pkg="fleet_adapter_for_kachaka" exec="fleet_adapter" args="--config_file $(var config_file) --nav_graph $(var nav_graph_file)" output="both">
  </node>

  <!-- delivery_node -->
  <node pkg="fleet_adapter_for_kachaka" exec="delivery_node" args="-c $(var config_file)" output="both">
  </node>

</launch>
